src/node_errors.cc

```diff
@@ -215,6 +215,12 @@ void AppendExceptionLine(Environment* env,
   Local<Object> err_obj;
   if (!er.IsEmpty() && er->IsObject()) {
     err_obj = er.As<Object>();
+    // If arrow_message is already set, skip.
+    auto maybe_value = err_obj->GetPrivate(env->context(),
+                                          env->arrow_message_private_symbol());
+    Local<Value> lvalue;
+    if (maybe_value.ToLocal(&lvalue) && lvalue->IsString())
+      return;
```

Comment:
I think you meant this?

```suggestion
    if (!maybe_value.ToLocal(&lvalue) || lvalue->IsString())
      return;
```

Reply:
Thanks for the review. It is actually meant as an early opt-out of this function for when it is a string.

Reply:
Yes, but if a `ToLocal()` call returns false, you must return asap as well.

Reply:
Ahh ok thanks.